src/XrdCeph/XrdCephOss.hh

```diff
@@ -65,12 +65,20 @@ public:
   virtual int     Rename(const char *, const char *, XrdOucEnv *eP1=0, XrdOucEnv *eP2=0);
   virtual int     Stat(const char *, struct stat *, int opts=0, XrdOucEnv *eP=0);
   virtual int     StatFS(const char *path, char *buff, int &blen, XrdOucEnv *eP=0);
+  virtual int     StatLS(XrdOucEnv &env, const char *path, char *buff, int &blen);  
   virtual int     StatVS(XrdOssVSInfo *sP, const char *sname=0, int updt=0);
   virtual int     Truncate(const char *, unsigned long long, XrdOucEnv *eP=0);
   virtual int     Unlink(const char *path, int Opts=0, XrdOucEnv *eP=0);
   virtual XrdOssDF *newDir(const char *tident);
   virtual XrdOssDF *newFile(const char *tident);
 
+  private:
```

Comment:
There seem to be some member variables which are from the bufferedIO branch, but I thought this would end up in master (first) which does not have the bufferedIO. 
Are these variables expected here?